src/pdsparsemat.jl

```diff
@@ -50,9 +48,6 @@ end
 
 Base.inv(a::PDSparseMat{T}) where {T<:Real} = PDMat( a\eye(T,a.dim) )
 LinearAlgebra.logdet(a::PDSparseMat) = logdet(a.chol)
-LinearAlgebra.eigmax(a::PDSparseMat{T}) where {T<:Real} = convert(T,eigs(convert(SparseMatrixCSC{Float64,Int},a.mat), which=:LM, nev=1, ritzvec=false)[1][1]) #to avoid type instability issues in eigs, see e.g., julia issue #13929
```

Comment:
Are there fallback definitions somewhere that will produce the right thing for a `PDSparseMat`?

Reply:
No. The PR is removing functionality based on a trade off between how useful this method is (I don’t think it is) and how costly it is to have the binary dependency for all downstream packages

Reply:
Okay. I don't have an opinion here so I trust your judgement on this as long as we make this a breaking release.